src/ott-analytics.js

```diff
@@ -14,7 +14,14 @@ const OttAnalyticsEvent: OttAnalyticsEventType = {
   HIT: 'HIT'
 };
 
-export default class OttAnalytics extends BasePlugin {
+const BookmarkError = {
+  BOOKMARK_ERROR: 'bookmarkerror',
+  CONCURRENCY_LIMIT: 'concurrencylimit'
+};
+
+const CONCURRENCY_ERROR_CODE: string = '4001';
```

Comment:
should we move this to our playkit err codes?

Reply:
no, these are plugin specific errors.
I'm thinking of exposing them as plugin constants on the module statics, so an app can do something like `KalturaPlayer.plugins.ottAnalytics.XYZ` where XYZ is either events or errors enum